{
    "osv": {
        "package": {
            "name": "python2.2"
        },
        "published": "2005-09-22",
        "references": [
            {
                "type": "DSA",
                "url": "https://www.debian.org/security/2005/dsa-817"
            }
        ]
    },
    "uvi": {
        "experimental": {
            "advisory_type": "DSA-817-1",
            "other_identifiers": {
                "cve": [
                    "CVE-2005-2491"
                ]
            },
            "package_affected": [
                {
                    "distribution_affected": "woody",
                    "fixed in": "2.2.1-4.8"
                },
                {
                    "distribution_affected": "sarge",
                    "fixed in": "2.2.3dfsg-2sarge1"
                },
                {
                    "distribution_affected": "sid",
                    "fixed in": "2.2.3dfsg-4"
                }
            ],
            "package_name": "python2.2",
            "processed_timestamp": "2021-07-18T03:08:15.867376Z",
            "uvi_script_name": "./www.debian.org-processing.py",
            "uvi_script_version": "0.0.5",
            "vulnerability_status": "yes"
        },
        "product_name": "Linux",
        "vendor_name": "Debian"
    }
}